training/unit-testing/go-lang/README.md

```diff
@@ -0,0 +1,126 @@
+Unit Testing in Go
+==================
+
+### Pre reading & Pre requisites
+It's advised that you are comfortable with GoLang or another programming or scripting language.  The setup is focussed on linux based operating systems..
```

Comment:
Should we rename GoLang to Go if not at least Golang (not sure why the L is capitalised here).

fyi - "The language is often referred to as Golang because of its domain name, golang.org , but the proper name is Go."